assets/javascripts/lib/discourse-markdown/bbcode-plugin.js

```diff
@@ -95,10 +95,16 @@ export function setup(helper) {
     "div.bb-spoiler-content",
     "div[style=*]",
     "details.bb-spoiler",
+    "h1.bb-header",
+    "h2.bb-header-2",
```

Comment:
@kyuukestu, clarification question here—if we aren't adding any additional styles to the h2-h6 tags, do we need to apply the class and whitelist it? Does not applying a class style cause any issues with just transforming those nodes into hX tags?

Reply:
I wasn't sure if it was a case that they would always be unstyled, so I added classes so they could be styled at a later date if desired. I don't believe there would be any issue with removing the class since I was able to create the scroll box without adding a class.

I can certainly remove the classes if that's preferred o7

Reply:
I'm fine with adding the classes. I'm also fine without. It shouldn't conflict with the existing styles for those headers.

Though if you do add the classes, simplify the class name to just `bb-header`. We can target specific heading levels with some tag specificity in the css anyway (i.e. `h2.bb-header`